dist/angular-modal-drawer.js

```diff
@@ -70,10 +74,15 @@
             }
           }
 
+          if(initOptions.disableParentScroll){
```

Comment:
Can we DRY mutiple if statements here ? (low priority)

Reply:
yes, will look over it